### DISCUSSION TITLE
Welcome to awesome-earthobservation-code Discussions!

### DISCUSSION THREAD

**Discussion by acgeospatial** [acgeospatial/awesome-earthobservation-code#17] [General]
## 👋 Welcome to awesome-earth-observation discussions
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

If you have any links to Earth Observation code this would be another good place to share them. Don't like a link? Tell us why and we will review it and give an honest response as to why we will keep or remove it.

My name is Andrew and I curate this list. You can learn more about me here [my website](http://www.acgeospatial.co.uk/). I am very pleased to meet you.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by EkicierNico** [comment]
Hello Andrew,
I'm Nicolas and I'm a remote sensing researcher. I really appreciate your repository, it's very nice to share this kind of list to the community. Thus, I would like to suggest you my own python module, there are some functions to process image or vector file.
Here is the link, do not hesitate to contact me to share your opinion or other things.
Thanks a lot

https://github.com/EkicierNico/pykic

**Reply by acgeospatial** [reply]
Thankyou, glad its helpful! I've added your module

**Comment by kempenep** [comment]
Hi Andrew,

Congratulations with this nice repository. Similar request here. I am with the Joint Research Centre (JRC) of the European Commission. We have developed a Python library, [pyjeo](https://github.com/ec-jrc/jeolib-pyjeo), that is [documented](https://jeodpp.jrc.ec.europa.eu/services/processing/pyjeohelp/) and might fit in your list as well. Its focus is on remote sensing image processing (raster and vector) and has been released as open source under the GPLv3 license. As part of the openEO consortium, we developed our in house openEO back-end on this python library. We took special care to bridge this library to other packages, including Numpy, xarray, and (geo)pandas.
Thanks,

Pieter.

**Reply by acgeospatial** [reply]
Hi Pieter, Thanks for those kind comments. Cool project! I've added it to the repor under python post processing. Thankyou!
Andrew
